R/mod-review-section.R

```diff
@@ -67,7 +67,7 @@ mod_review_section_ui <- function(id) {
           inputId = ns("section_species"),
           label = "Species",
           choices = c(
-            "Not applicable" = NA,
+            "Not applicable" = "",
```

Comment:
Not sure what the impact of this change will be? Does this change how we store the NA option on the backend?

Reply:
I just set this back - you're right, we shouldn't change the backend storage format. I originally did this because it makes updating the value of the field based on the backend data easier, but it still works with NA so better to leave as is. Thanks!